packages/cache-handler/src/handlers/redis-stack.ts

```diff
@@ -156,7 +156,7 @@ export default async function createHandler({
                     },
                 );
 
-                for (const { id } of documents) {
+                for (const id of documentIds) {
```

Comment:
Please change name here to match the name from above:

```suggestion
                for (const id of documents) {
```